<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyDialog</class>
 <widget class="QDialog" name="MyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>336</height>
   </rect>
  </property>
  <widget class="QWidget" name="controls_widget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>9</y>
     <width>301</width>
     <height>55</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>55</height>
    </size>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Nationality:</string>
    </property>
    <property name="buddy">
     <cstring>cbo_nationality</cstring>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_ordnance">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Ordnance</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbo_nationality">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_vehicles">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Vehicles</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="buttons_widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>361</width>
     <height>46</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="ok_button">
      <property name="text">
       <string>&amp;OK</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_button">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>53</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>&amp;Cards:</string>
   </property>
   <property name="buddy">
    <cstring>lb_cards</cstring>
   </property>
  </widget>
  <widget class="QListWidget" name="lb_cards">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>71</y>
     <width>362</width>
     <height>211</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
